src/main/java/StringCalculator.java

```diff
@@ -0,0 +1,83 @@
+/*
+input을 받고 빈문자열인지 null인지 체크
+구분자 체크 -> 구분자 예외 체크
+구분자 대로 문자열 계산기 구현
+ */
+
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    //입력 문자열이 null인지 확인
+    public String checkInput(String input){
+        if(input == null) throw new RuntimeException("null은 허용되지 않습니다.");
+        return input;
+    }
```

Comment:
checkInput메소드에서 결과를 반환하지 않아도 될 것 같습니다!
```suggestion
    public void validateInput(String input){
        if(input == null) {
            throw new RuntimeException("null은 허용되지 않습니다.");
        }
    }
```

Reply:
그렇네요... 확실히 input을 반환하지 않고 잘못된 입력이면 예외를 발생하면 될 것 같습니다!